### PR TITLE
Make detailed logs optional.

### PR DESCRIPTION
**Pull request by keithjjones**
This will also turn off suppression on notices since that is the only place each C2 message would be logged when detailed logging is turned off.

### CODE REVIEW

File: scripts/main.zeek [hunk 1]
```diff
@@ -26,52 +29,49 @@ export {
 
 	## A default logging policy hook for the stream.
 	global log_policy: Log::PolicyHook;
+
+	## Indicator of a request related to GOZI
+	redef enum HTTP::Tags += { URL_GOZIMALWARE, };
```
Comment (pauldokas) [inline scripts/main.zeek]: Following the SQLI pattern, I suggest that this should be 'URI_GOZIMALWARE'.

Comment (keithjjones) [reply scripts/main.zeek]: Done.

File: scripts/main.zeek [hunk 2]
```diff
@@ -26,52 +29,49 @@ export {
...
-	{
-	if ( c?$gozi )
-		return;
-
-	c$gozi = Info($ts=network_time(), $uid=c$uid, $id=c$id);
-	}
-
-function log_gozi_detected(c: connection)
+function log_gozi_detected(c: connection, http_method: string, payload: string,
+    enable_detailed_logs: bool)
```
Comment (pauldokas) [inline scripts/main.zeek]: Is it necessary to pass in `enabled_detailed_logs` since it's scope is the package?

Comment (keithjjones) [reply scripts/main.zeek]: Fixed

**Pull request merged**